imported/spirv/spvgen.h

```diff
@@ -421,10 +419,19 @@ bool InitSpvGen()
     }
 
     bool success = true;
+    const char* pLibName = SpvGeneratorName;
+    std::string libNameBuffer;
+    if (pSpvGenDir != nullptr)
+    {
+        libNameBuffer = pSpvGenDir;
+        libNameBuffer += "/";
+        libNameBuffer += pLibName;
+        pLibName = libNameBuffer.c_str();
+    }
 #ifdef _WIN32
-    HMODULE hModule = LoadLibrary(SpvGeneratorName);
+    HMODULE hModule = LoadLibrary(pLibName);
```

Comment:
please replace LoadLibrary with LoadLibraryA, otherwise, build will fail when UNICODE is defined.

Reply:
Good point. I made that change in a different internal version of the patch, but I forgot to transfer it back to this version.